src/app/market/buy/checkout-process/checkout-process.component.ts

```diff
@@ -113,14 +129,16 @@ export class CheckoutProcessComponent implements OnInit, OnDestroy {
   }
 
   removeFromCart(shoppingCartId: number): void {
-    this.cartService.removeItem(shoppingCartId).take(1).subscribe();
+    this.cartService.removeItem(shoppingCartId).take(1).subscribe(() => this.setCartItemCount());
   }
 
   clearCart(isSnack: boolean = true): void {
     this.cartService.clear().subscribe(res => {
       if (isSnack) {
         this.snackbarService.open('All Items Cleared From Cart');
       }
+      this.resetStepper();
+      this.setCartItemCount();
```

Comment:
is this required, clearing the cart will trigger an update() and it will already call this.setCartItemCount() in the subscription.